Dockerfile

```diff
@@ -10,4 +10,9 @@ RUN conda install pyqt -y && \
     pip install -U pytest && \
     pip uninstall h5py && \
     pip install --no-cache-dir h5py && \
-    python setup.py install
+    python setup.py install && \
+    sudo apt-get update && \
```

Comment:
Why update so many times?

Reply:
Not sure. That's what the person did on stackoverflow (https://stackoverflow.com/questions/48965309/h5py-import-error-on-libhdf5-serial-so-100)

Reply:
interesting.